### ISSUE TITLE
How to automatically generate a PR to resolve an application issue?

### ISSUE THREAD

**Issue by hackfisher** [dcdao/profiles#14]
This PR should be automatically linked to the application issue and the issue will be closed once the PR is approved and merged.

**Comment by hackfisher** [comment]
Example:

https://github.com/dcdao/profiles/pull/10

https://github.com/dcdao/profiles/issues/7

**Comment by furoxr** [comment]
I think it's better to generate a PR manually, and let the CI to commit files for application issues. The current workflow is designed to accept a batch of applications and then mint them on the blockchain, utilizing the content in the release.md file. When we're ready to accept applications, we create a release branch and empty the release.md file. After the application period is over, we proceed to mint NFTs based on the data in the release.md file.

The feature to link issues to the PR is nice to have, I could implement it in next version.
